sys/include/net/netopt.h

```diff
@@ -195,6 +195,12 @@ typedef enum {
      * @brief   (uint16_t) maximum protocol data unit
      */
     NETOPT_MAX_PDU_SIZE,
+    /**
+     * @brief   (uint16_t) protocol data unit
+     *
+     * Get/Set the PDU size on link layers with implicit header (e.g LoRaWAN).
```

Comment:
Does this mean the payload will be padded if it is smaller than this size?

Reply:
yes but probably with garbage data

Reply:
That's fine, but it needs to be documented :wink:

Reply:
@fabian18 mentioned something similar is needed on nrf24l01 (non-Plus)

Reply:
done!